com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -380,8 +380,20 @@ public static void NetworkHide(List<NetworkObject> networkObjects, ulong clientI
             }
         }
 
+        private bool m_ApplicationQuitting = false;
```

Comment:
if you wanted to avoid the instance memory, you could do this statically:

```
static class ApplicationQuitHelper
{
    public static bool ApplicationQuitting { get; private set; }

    [RuntimeInitializeOnLoadMethod]
    static void RegisterQuitCallback()
    {
        Application.quitting += ()=> ApplicationQuitting = true;
    }
}
```